### DISCUSSION TITLE
Welcome to Input Discussions!

### DISCUSSION THREAD

**Discussion by PhilReinking** [deck9/input#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by candideu** [comment]
Hello! I discovered your app on Alternativeto, and really love that there's an open source version of Typeform! 

I just wish that there was a demo on your website landing page of what a form looks like (see YouForm: https://youform.com/)

**Reply by PhilReinking** [reply]
Thanks @candideu for the suggestion. This is indeed something we are working on along with a new user documentation and some demo examples of how Input can be used. I plan on updating the landing page in the coming weeks!
